app/configurator/components/field.tsx

```diff
@@ -171,25 +171,33 @@ export const DataFilterSelect = ({
     message: `optional`,
   });
 
-  const allOptions = useMemo(() => {
+  const sortedValues = useMemo(() => {
+    const sorters = makeDimensionValueSorters(dimension);
+    const sortedValues = orderBy(dimension.values, sorters);
+
+    return sortedValues;
+  }, [dimension]);
+
+  const allValues = useMemo(() => {
```

Comment:
I wonder if here we shouldn't apply the same, fixed approach as in hierarchy tree sorting ["position", "identifier", "label"]. Not sure if values in filters should be affected by chart sorting, e.g. by measure.

Reply:
Yes it makes sense that it should not be dependent on the legend display, as as a chart publisher you'd want to always see the same order 👍 Isn't it the case here since we do not pass a sorting to makeDimensionValueSorters ? Seems to me that in this case we should default to the defaultSorters  ?

```
  const defaultSorters = [getPosition, getIdentifier, getLabel];
```

Reply:
Yes, I just realised that 👍